q2_phylogeny/plugin_setup.py

```diff
@@ -344,3 +344,57 @@
     description=("Remove features from a feature table if their identifiers "
                  "are not tip identifiers in tree.")
 )
+
+plugin.pipelines.register_function(
+    function=q2_phylogeny.fasttree_with_mafft,
+    inputs={
+        'sequences': FeatureData[Sequence],
+    },
+    parameters={
+        'n_threads': Int % Range(1, None),
+        'max_gap_frequency': Float % Range(0, 1, inclusive_end=True),
+        'min_conservation': Float % Range(0, 1, inclusive_end=True)
+    },
+    outputs=[
+        ('alignment', FeatureData[AlignedSequence]),
+        ('masked_alignment', FeatureData[AlignedSequence]),
+        ('tree', Phylogeny[Unrooted]),
+        ('rooted_tree', Phylogeny[Rooted]),
+    ],
+    input_descriptions={
+        'sequences': 'The sequences to be used for creating a '
+                     'fasttree based rooted phylogenetic tree.'
+    },
+    parameter_descriptions={
+        'n_threads': 'The number of threads. (Use -1 to automatically use all '
+                     'available cores)',
+        'max_gap_frequency':  'The maximum relative frequency of gap '
+                              'characters in a column for the column to be '
+                              'retained. This relative frequency must be a '
+                              'number between 0.0 and 1.0 (inclusive), where '
+                              '0.0 retains only those columns without gap '
+                              'characters, and 1.0 retains all columns '
+                              'regardless of gap character frequency.',
+        'min_conservation':  'The minimum relative frequency '
+                             'of at least one non-gap character in a '
+                             'column for that column to be retained. This '
+                             'relative frequency must be a number between 0.0 '
+                             'and 1.0 (inclusive). For example, if a value of '
+                             '0.4 is provided, a column will only be retained '
+                             'if it contains at least one character that is '
+                             'present in at least 40% of the sequences.'
+    },
+    output_descriptions={
+        'alignment': 'The aligned sequences.',
+        'masked_alignment': 'The masked alignment.',
+        'tree': 'The unrooted phylogenetic tree.',
+        'rooted_tree': 'The rooted phylogenetic tree.',
+    },
+    name='Construct a phylogenetic tree using mafft and fasttree',
```

Comment:
"Align sequences with mafft and build a phylogenetic tree with fasttree"
or
"Build a phylogenetic tree using fasttree and mafft alignment"